NineChronicles.Headless.Tests/SetAvatarState.cs

```diff
@@ -59,7 +59,8 @@ public override IAccountStateDelta Execute(IActionContext context)
 
```

Comment:
The changes of tests in this pull request became not related to `BlockChain<T>`. So it seems not related to also `SetAvatarState` action to me, doesn't it?